group_vars/winthrop_prod/vars.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+# These variables are the stock configurations that either alias variables
+# from the vault or provide paths that the project itself uses.
+# vault variables first
+secret_key: '{{ vault_secret_key }}'
```

Comment:
Could some of these also go in common variables? ansible will still pick up the correct vault file, right?

Reply:
Secret keys are actually unique to each instance between QA and PROD (or should be... in theory staging could piggy pack off QA), but deliberately we don't use the same secret keys so that if we have an error, we can limit damage.

Reply:
However, some of the others are common as I realized while working on Derrida. Will adjust accordingly.